docs/gno-tooling/cli/gnokey/querying-a-network.md

```diff
@@ -0,0 +1,199 @@
+---
+id: querying-a-network
+---
+
+# Querying a gno.land network
+
+## Prerequisites
+
+- **`gnokey` installed.** Reference the
+  [Local Setup](../../../getting-started/local-setup/installation.md#2-installing-the-required-tools) guide for steps
+
+## Overview
+
+gno.land and `gnokey` support ABCI queries. Using ABCI queries, you can query the state of 
+a gno.land network without spending any gas. All queries need to be pointed towards 
+a specific remote address from which the state will be retrieved.
```

Comment:
Can we add a brief explanation of what an ABCI query is? And maybe a link to an external resource for more details?

Why don’t I spend any gas while using them (in a couple of words)? Is it because I'm just reading the state without mutating it?

Reply:
Personally, when I had the exact same question while reading the Gno docs, I ended up checking out the [Tendermint docs](https://docs.tendermint.com/v0.33/app-dev/app-development.html) on the topic. 
But it's a bit hard to digest for anyone who just wants to quickly understand what it's about and what the implications are for them as a user.

Reply:
Agree. The issue is that this is a cosmos term, not gno.land specific. We could have a doc page on this though.

Reply:
Yes, I don't know if it's out of scope or not, it's maybe better to close this point and address it in another PR?